ecosystem/sep-0018.md

```diff
@@ -50,6 +50,20 @@ wallet.eth
 * **Term REGEXP:** `[_a-z][_a-z0-9]*`
 * **Key REGEXP:** `[_a-z][_a-z0-9]*(\.[_a-z][_a-z0-9]*)*`
 
+## Namespace Registry
+
+To guarantee interoperability and avoid naming conflicts, namespaces must be
+used as defined in SEP documents.
+
+- `app`: Space dedicated to vendor-specific keys. For example, "example.org" can
+  define its keys under `app.example_org`.
```

Comment:
👍 I like that the `app` namespace has an example of how an app should use the namespace by specifying a sub-namespace that is a domain replacing `.`s with `_`. That seems simple and straight forward. Could we specify this?

With the 64-byte data entry key length limitation only apps with domains less than 60 characters in length will be supported. That's probably fine given 60 characters is very long and most websites or apps don't have domains that are that long.

Reply:
> Could we specify this?
Done